src/main/drivers/barometer/barometer_bmp085.c

```diff
@@ -174,14 +173,13 @@ bool bmp085Detect(const bmp085Config_t *config, baroDev_t *baro)
     bmp085InitXclrIO(config);
     BMP085_ON;   // enable baro
 
-#if defined(BARO_EOC_GPIO) && defined(USE_EXTI)
+#if defined(BARO_EOC_PIN) && defined(USE_EXTI)
     if (config && config->eocIO) {
         eocIO = IOGetByTag(config->eocIO);
         // EXTI interrupt for barometer EOC
         IOInit(eocIO, OWNER_BARO_EXTI, 0);
-        IOConfigGPIO(eocIO, Mode_IN_FLOATING);
```

Comment:
Why was this removed?

Reply:
look two lines below, `EXTIConfig` does the `IOConfigGPIO` work.